admin/github.go

```diff
@@ -54,12 +70,27 @@ func NewGithub(appID int64, appPrivateKey string) (Github, error) {
 		panic(err)
 	}
 
-	return &githubClient{
+	g := &githubClient{
 		appID:             appID,
 		appPrivateKey:     appPrivateKey,
 		appClient:         appClient,
 		installationCache: lru,
-	}, nil
+		managedAcct:       githubManagedAcct,
+	}
+
+	// Set the managed org installation client
+	if githubManagedAcct == "" {
+		g.managedOrgFetchError = fmt.Errorf("managed Git repositories are not configured for this environment")
+	}
+	i, _, err := appClient.Apps.FindOrganizationInstallation(ctx, githubManagedAcct)
+	if err != nil {
+		logger.Error("failed to get managed org installation ID", zap.Error(err), observability.ZapCtx(ctx))
+		g.managedOrgFetchError = err
+		return g, nil
+	}
```

Comment:
Don't you need an `else` here? Otherwise `g.managedOrgFetchError` would always get overridden with the Github error for empty repo name?